src/styles.css

```diff
@@ -0,0 +1,120 @@
+.app {
```

Comment:
I don't know much about CSS (I don't want to analyze it), but doesn't look good on mobile

Reply:
so you could parametrize your CSS using media queries - https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries